pkg/apis/net/v1alpha1/poolservice_types.go

```diff
@@ -26,12 +26,6 @@ import (
 
 // PoolServiceSpec defines the desired state of PoolService
 type PoolServiceSpec struct {
-	// Specify the name of the nodepool to identifies the nodepool to which the PoolService belongs.
-	PoolName string `json:"poolName,omitempty"`
```

Comment:
@zyjhtangtang Without the poolName in the spec, how do we identify the poolservice under different nodepools?

Reply:
PoolServices will now be identified using a label: openyurt.io/pool-name. Additionally, the nodepool will also serve as the ownreferences for the poolservice.